__SIGNAL_FRAMESIZE is defined in asm/sigcontext.h but including it would make
conflict with struct sigcontext since we compile with it in musl. This patch
is temporary till musl upstream has a proper solution for sigcontext.

diff --git a/linux/s390/arch_sigreturn.c b/linux/s390/arch_sigreturn.c
index d1b8598..c0a4658 100644
--- a/linux/s390/arch_sigreturn.c
+++ b/linux/s390/arch_sigreturn.c
@@ -2,7 +2,7 @@ static void
 arch_sigreturn(struct tcb *tcp)
 {
 	unsigned long mask[NSIG_BYTES / sizeof(long)];
-	const unsigned long addr = *s390_frame_ptr + __SIGNAL_FRAMESIZE;
+	const unsigned long addr = *s390_frame_ptr + 160;
 
 	if (umove(tcp, addr, &mask) < 0) {
 		tprintf("{mask=%#lx}", addr);
